dev-infrastructure/docs/development-setup.md

```diff
@@ -344,7 +357,13 @@ instance_types:
 EOF
 ```
 
-3) Follow CS dev setup process:
+3) Get first-party application certificate bundle (Note that this will change when new FPA changes are introduced soon): </br>
+Run the following command to generate a file containing the base64 decoded first-party application certificate bundle
+```shell
+$ az keyvault secret download --file fpa_cert --id=https://service-kv-aro-hcp-dev.vault.azure.net/secrets/firstPartyMock/<certi_id> | base64 -d > ~/fpa_cert_decoded
```

Comment:
certi_id -> cert_id.

I think we might be able to make it easier to get the secret URL associated to the certficate than the user having to know the certificate id. cc @JameelB @janboll could you take a look?

Thanks

Reply:
+1

alternatively, we could also just specify the vault name and name like `az keyvault secret download --vault-name "aro-hcp-dev-svc-kv" --name "firstPartyCert" -f`

Since these values are already used in the cs makefile, we could add a target for doing this. Doesn't have to be in this PR but would be nice in the future.